bindings/go/cel/expression/parser/cel.go

```diff
@@ -0,0 +1,124 @@
+package parser
+
+import (
+	"errors"
+	"strings"
+)
+
+const (
+	// CEL expressions are enclosed between "${" and "}"
+	exprStart = "${"
```

Comment:
this comes from kro right?

https://github.com/open-component-model/open-component-model/blob/24b8f8099d9e5e74c2aeec4b898b72374da04f98/docs/adr/0005_transformation.md?plain=1#L13

wouldn't it be easier to go with go-templates if we had other characters to wrap the expressions?

Reply:
go templates are dynamic and a DSL, cel is a full expression language that is type safe by default.